src/Explorer/Notebook/NotebookRenderer/outputs/IFrameOutputs.tsx

```diff
@@ -0,0 +1,103 @@
+import { AppState, ContentRef, selectors } from "@nteract/core";
+import { Output } from "@nteract/outputs";
+import Immutable from "immutable";
+import React from "react";
+import ReactDOM from "react-dom";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import * as _ from "underscore";
+import { copyStyles } from "../../../../Utils/StyleUtils";
+
+// Adapted from https://github.com/nteract/nteract/blob/main/packages/stateful-components/src/outputs/index.tsx
+// to add support for sandboxing using <iframe>
+
+interface ComponentProps {
+  id: string;
+  contentRef: ContentRef;
+  children: React.ReactNode;
+}
+
+interface StateProps {
+  hidden: boolean;
+  expanded: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  outputs: Immutable.List<any>;
```

Comment:
does `unknown` work here instead of `any`?

Reply:
Unfortunately `unknown` doesn't work here. This component uses `<Output />` which relies on `outputs` with this same exact signature.